pkg/simple/client/devops/jenkins/pure_request.go

```diff
@@ -24,10 +30,41 @@ func (j *Jenkins) SendPureRequestWithHeaderResp(path string, httpParameters *dev
 
 	client := &http.Client{Timeout: 30 * time.Second}
 
+	header := httpParameters.Header
+	bearTokenArray := strings.Split(header.Get("Authorization"), " ")
+	bearFlag := bearTokenArray[0]
+	if err == nil && bearFlag == "Bearer" {
+		bearToken := bearTokenArray[1]
+		if err != nil {
+			klog.Error(err)
+			return nil, nil, err
+		}
+		infoArray := strings.Split(bearToken, ".")
```

Comment:
Why not using a tokenIssuer `pkg/apiserver/authentication/token/jwt.go` to parse a token?

Reply:
fixed